sdk/subscribe_block.go

```diff
@@ -66,6 +69,7 @@ func (c *SubscribeService) UnconfirmedAdded(add string) (*SubscribeTransaction,
 // address was in unconfirmed state but not anymore.
 // The message contains the transaction hash.
 func (c *SubscribeService) UnconfirmedRemoved(add string) (*SubscribeHash, error) {
```

Comment:
Add please Address struct as param to subscribe methods

Reply:
With what intention! I only use the address of the account?